zipkin-lens/src/components/UiConfig/index.js

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright 2015-2020 The OpenZipkin Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+export * from './UiConfig';
```

Comment:
do you love our license header?

Reply:
Nope. I think separating `index.js` and `Component.jsx` is not useful, but noticed the current webpack config doesn't seem to allow `index.jsx`. So will fix that in another PR :)